signer/sign_blinded_beacon_block_test.go

```diff
@@ -0,0 +1,60 @@
+package signer
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/attestantio/go-eth2-client/api"
+	apiv1bellatrix "github.com/attestantio/go-eth2-client/api/v1/bellatrix"
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/stretchr/testify/require"
+
+	eth2keymanager "github.com/bloxapp/eth2-key-manager"
+	"github.com/bloxapp/eth2-key-manager/core"
+	prot "github.com/bloxapp/eth2-key-manager/slashing_protection"
+	"github.com/bloxapp/eth2-key-manager/wallets"
+)
+
+// tested against a ...
+func TestBenchmarkBlindedBlockProposalBellatrix(t *testing.T) {
```

Comment:
add a test for signing a block and then a blinded block (and vice versa). It should fail on the second sig